### PR TITLE
Testing upgrades

### PR DESCRIPTION
**Pull request by markbudos**

### CODE REVIEW

File: cardsavr_e2e/CardSavrHelperTests.cs [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            //login can now be used as a redirect to a url that can log in and process jobs
+        }
+
+        public override async Task Cleanup(CardSavrHttpClient http, Context ctx)
+        {
+            await base.Cleanup(http, ctx);
+        }
+
+    }
+}*/
```
Comment (squann) [inline cardsavr_e2e/CardSavrHelperTests.cs]: What's this?

Comment (markbudos) [reply cardsavr_e2e/CardSavrHelperTests.cs]: It's basically exactly what Advancial does.  (In fact, it's all that they do)

Comment (markbudos) [reply cardsavr_e2e/CardSavrHelperTests.cs]: But now I'm looking, the class should be renamed.  The way the framework is, the filename is irrelevant.

Comment (markbudos) [reply cardsavr_e2e/CardSavrHelperTests.cs]: Oh I see, you're talking about the commented out code.  That's the old test, I just removed it.

**Review by squann — APPROVED** [review]

**Pull request merged**